boilerplate/openshift/golang-osd-operator/standard.mk

```
@@ -46,6 +46,9 @@ OPERATOR_IMAGE_URI=${IMG}
 OPERATOR_IMAGE_URI_LATEST=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME):latest
 OPERATOR_DOCKERFILE ?=build/Dockerfile
 REGISTRY_IMAGE=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME)-registry
+#The api dir that latest osdk generated
+NEW_API_DIR=$(PWD)/api
```

I think `$(PWD)` is coming from your env, and may or may not be what you expect. This would probably work well as
```suggestion
NEW_API_DIR=./api
```

Similarly for the other places you're using `$(PWD)`.

If you actually do want to use the full path, consider evaluating it on the fly, like
```makefile
PWD := $(shell pwd)
```

---

I did something similar earlier today, forgot to update it here sorry. That did the trick, works now. Thanks!

---

thanks, ./api works.